examples/z_get.c

```diff
@@ -14,37 +14,27 @@
 #include <stdio.h>
 #include <string.h>
 
+#include "parse_args.h"
 #include "zenoh.h"
 
-int main(int argc, char **argv) {
-    char *expr = "demo/example/**";
-    char *value = NULL;
-    switch (argc) {
-        default:
-        case 3:
-            value = argv[2];
-        case 2:
-            expr = argv[1];
-            break;
-        case 1:
-            // Do nothing
-            break;
-    }
-    z_keyexpr_t keyexpr = z_keyexpr(expr);
+#define DEFAULT_SELECTOR "demo/example/**"
+#define DEFAULT_VALUE NULL
+
+struct args_t {
+    char* selector;  // -s
+    char* value;     // -v
+};
+struct args_t parse_args(int argc, char** argv, z_owned_config_t* config);
```

Comment:
You should provide some free method for this structure and cleanup it, before exit. This is a good practice, and the example should show good practice, many users simply copy the code from the examples and the program does not always exit immediately after using the example, the user will have a memory leak.